consumer/init_consumer.py

```diff
@@ -1,35 +1,42 @@
 import os
-#import boto3
+import boto3
 
 from consumer import Consumer
 from kafka_message.ttypes import KafkaMessage
 from thrift.TSerialization import deserialize
 from cassandra.cluster import Cluster
 
 
-def callback(msg, session):
-    kafka_message = KafkaMessage()
-    deserialize(kafka_message, msg.value)
-    print kafka_message
-    
-    #client = boto3.client("sns")
+def send_msg(client, contact, text):
+    print client.publish(
+        TopicArn="test-topic", 
+        Message=text
+        )
+
+def get_user_data(uid, session):
+        user_rows = session.execute("SELECT * FROM users WHERE uid='%s'" % (uid))
 
-    m_rows = session.execute("SELECT * FROM messages WHERE mid='%s'" % (kafka_message.mid))
+        for user_row in user_rows:
+            return user_row
 
-    for m_row in m_rows:
-        print m_row
+def get_message_data(mid, session):
+    msg_rows = session.execute("SELECT * FROM messages WHERE mid='%s'" % (mid))
 
-        u_rows = session.execute("SELECT * FROM users WHERE uid='%s'" % (m_row.uid))
+    for msg_row in msg_rows:
+        return msg_row
 
-        for u_row in u_rows:
-            print u_row
+def deserialize_msg(msg):
+    kafka_message = KafkaMessage()
+    deserialize(kafka_message, msg.value)
+    return kafka_message
 
+def kafka_message_handler(session, sns_client, kafka_msg):
+    kafka_msg = deserialize_msg(kafka_msg)
+    msg = get_message_data(kafka_msg.mid, session)
+    user = get_user_data(msg.uid, session)
+    print "To: %s. Message: '%s'." % (user.contact, msg.text)
+    #send_msg(sns_client, user.contact, msg.text)
```

Comment:
Please NEVER leave commented code - instead make `send_msg` log message to console instead of sending it via SNS